### DISCUSSION TITLE
Slideouts in entry index

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#9776] [Ideas] [answered]
Slideouts are a great UI feature since 3.7.

I think that adding an option to open an entry in "slideout mode" from the entry index page (e.g. there could be an "open in slideout" optional column with a button/link) would be a great way to facilitate quick edits without losing context.

Would anyone else find this useful or is it just me?

**Comment by mmikkel** [comment]
I think a "Quick Edit" button in the index is a really good idea, especially as an accessibility feature in addition to the existing double-click functionality.

**Comment by brandonkelly** [comment, marked as answer]
Craft already supports slideouts on the Entries index page.

You can open them in two ways:

- Double-click on an entry’s status icon
- Select an entry’s checkbox, and then click “Edit entry” from the action menu.

<img src="https://user-images.githubusercontent.com/47792/131527413-f9450bf6-8fa1-4b6f-8efc-a0b4f7578011.png" width="360" alt="The action menu for a selected entry on the Entries index page, highlighting the “Edit entry” option">

You can also double-click the status icon/thumbnail of any related elements that have a visible column in the table.

**Reply by thupsi** [reply]
I admit I was aware only of ability to open related elements as you describe. The discoverability of the other methods is not great though and the status icon is a bit on the small side as a click target, so IMO it does pose an accessibility concern. Other than that, it seems that my wish is already granted!

**Reply by brandonkelly** [reply]
I’d say it’s more of a power user feature rather than essential UI, and therefore not worth adding additional UI complexity over.

**Reply by thupsi** [reply]
Fair enough!
